Dockerfile-linux.template

```diff
@@ -0,0 +1,139 @@
+{{
+	def version_id:
+		# https://www.php.net/phpversion
+		# $version_id = $major_version * 10000 + $minor_version * 100 + $release_version;
+		sub("[a-zA-Z].*$"; "")
+		| split(".")
+		| (
+			(.[0] // 0 | tonumber) * 10000
+			+ (.[1] // 0 | tonumber) * 100
+			+ (.[2] // 0 | tonumber)
+		)
+	;
+	def is_alpine:
+		env.from | startswith("alpine")
+-}}
+FROM {{ env.from }}
+
+# Persistent dependencies
+RUN set -eux; \
+    apk add --no-cache \
+# Ghostscript is required for rendering PDF previews
+        ghostscript \
+        tzdata \
+        curl \
+        ca-certificates \
+        supervisor \
+    ;
+
+### Setup needed PHP dependencies ###
+# Install the PHP extensions we need (https://make.wordpress.org/hosting/handbook/handbook/server-environment/#php-extensions)
+RUN set -ex; \
```

Comment:
Can we combine the three `RUN` commands into one `RUN` command? This reduces the overall image size as there would only be one layer instead of three. Just keep comments separating the sections for sanity 😉

Reply:
Bonus points if you can get it all down to a single RUN command 😉

Reply:
Done.